Taskfile.yml

```diff
@@ -273,6 +273,19 @@ tasks:
         --tigris-url=127.0.0.1:8081
         --test-records-dir=tmp/records
 
+  run-hana:
+    deps: [build-local]
+    dotenv: ['.env', '{{.ENV}}/.env.', '{{.HOME}}/.env'] # Create a .env file in root folder with: export hana_url=”insert-url-here”
```

Comment:
We recommend contributors to use direnv, see https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md. But we also should not rely on particular tools and files they use.

In this case, FerretDB will handle `FERRETDB_HANA_URL` environment variable automatically, so there is no need to pass `--hana-url` flag.

Reply:
Ahh okay, super! I was not aware of this being possible. How do you do this? I cannot find FERRETDB_HANA_URL any place in the code? Just asking out of interest 😊

Reply:
`kong.DefaultEnvars` does that: https://pkg.go.dev/github.com/alecthomas/kong#DefaultEnvars

Reply:
Thank you!